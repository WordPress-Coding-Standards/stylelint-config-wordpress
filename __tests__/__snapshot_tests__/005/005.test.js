'use strict';

const stylelint = require( 'stylelint' ),
	systemTestUtils = require( '../systemTestUtils.js' );

it( '005', () => {
	return stylelint.lint({
		files: [systemTestUtils.caseStylesheetGlob( '005' )],
		configFile: systemTestUtils.caseConfig( '005' ),
	}).then( ({ results }) => {
		expect( systemTestUtils.prepResults( results ) ).toMatchSnapshot();
	});
});
