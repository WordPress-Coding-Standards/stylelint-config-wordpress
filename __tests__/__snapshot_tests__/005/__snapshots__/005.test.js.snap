// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`005 1`] = `
Array [
  Object {
    "deprecations": Array [],
    "errored": true,
    "ignored": undefined,
    "invalidOptionWarnings": Array [],
    "parseErrors": Array [],
    "source": "005/stylesheet.scss",
    "warnings": Array [
      Object {
        "column": 3,
        "line": 51,
        "rule": "at-rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before at-rule (at-rule-empty-line-before)",
      },
      Object {
        "column": 23,
        "line": 377,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace after \\"(\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 44,
        "line": 377,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace before \\")\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 5,
        "line": 188,
        "rule": "indentation",
        "severity": "error",
        "text": "Expected indentation of 1 tab (indentation)",
      },
      Object {
        "column": 5,
        "line": 231,
        "rule": "indentation",
        "severity": "error",
        "text": "Expected indentation of 1 tab (indentation)",
      },
      Object {
        "column": 5,
        "line": 236,
        "rule": "indentation",
        "severity": "error",
        "text": "Expected indentation of 1 tab (indentation)",
      },
      Object {
        "column": 83,
        "line": 133,
        "rule": "max-line-length",
        "severity": "error",
        "text": "Expected line length to be no more than 80 characters (max-line-length)",
      },
      Object {
        "column": 2,
        "line": 50,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 58,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 66,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 74,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 12,
        "line": 33,
        "rule": "selector-attribute-quotes",
        "severity": "error",
        "text": "Expected quotes around \\"checkbox\\" (selector-attribute-quotes)",
      },
      Object {
        "column": 12,
        "line": 37,
        "rule": "selector-attribute-quotes",
        "severity": "error",
        "text": "Expected quotes around \\"radio\\" (selector-attribute-quotes)",
      },
      Object {
        "column": 9,
        "line": 168,
        "rule": "selector-list-comma-newline-after",
        "severity": "error",
        "text": "Expected newline after \\",\\" (selector-list-comma-newline-after)",
      },
      Object {
        "column": 9,
        "line": 181,
        "rule": "selector-list-comma-newline-after",
        "severity": "error",
        "text": "Expected newline after \\",\\" (selector-list-comma-newline-after)",
      },
      Object {
        "column": 9,
        "line": 2,
        "rule": "string-quotes",
        "severity": "error",
        "text": "Expected double quotes (string-quotes)",
      },
      Object {
        "column": 9,
        "line": 3,
        "rule": "string-quotes",
        "severity": "error",
        "text": "Expected double quotes (string-quotes)",
      },
    ],
  },
]
`;
